crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -399,9 +458,17 @@ impl<'w, 's> Commands<'w, 's> {
 
     /// Returns the [`EntityCommands`] for the requested [`Entity`].
     ///
-    /// # Panics
+    /// This method does not guarantee that `EntityCommands` will be successfully applied,
+    /// since another command in the queue may delete the entity before them.
```

Comment:
What is "them"?

Reply:
Does this read better?

> This method does not guarantee that commands queued by the `EntityCommands` will be successful, since the entity could be despawned before they are executed.

Reply:
Yes